app/src/app/list/list.component.html

```diff
@@ -1,9 +1,27 @@
 <app-server-error [error]="error$ | async"></app-server-error>
-<div class="poll-list" *ngIf="(publicList$ | async)?.length">
-  <h3>Polls you participate in</h3>
-  <app-polls-table [polls]="publicList$ | async"></app-polls-table>
+<div id="mat-tab-container">
+  <mat-tab-group mat-align-tabs="center">
+    <mat-tab  label="Participating">
+      <div class="poll-list" *ngIf="(publicList$ | async)?.length">
```

Comment:
This div seems useless to me.

Reply:
I kept the div to have a gap between the tabs and the poll lists.

Reply:
A simple solution could be to simply add margin to h3.